src/04-modality-specific-files/29-motion.md

```diff
@@ -0,0 +1,224 @@
+
+## Channels description (`*_channels.tsv`)
+
+```markdown
+└─ sub-<label>\
+  └─ \[ses-<label>]\
+    └─ motion\
+      └─ sub-<label>\[\_ses-<label>][\_task-<label>\]_channels.tsv
+```
+
+This file is REQUIRED as it makes it easy to browse or query over larger collections of datasets. The REQUIRED columns are channel  `component`, `name`,  `tracked_point`, `type` and `unit`. Any number of additional columns may be added to provide additional information about the channels. The `*tracksys-<label>_channels.tsv` file should give additional information about individual recorded channel, some of which my not be found summarized in `*motion.json`.
+
+The columns of the channels description table stored in `*_channels.tsv` are:
+
+MUST be present:
+
+| **Key name**  | **Requirement level** | **Data type** | **Description**                                                                                                                                                                                |
+| ------------- | --------------------- | ------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| component     | REQUIRED              | string        | Component of the representational system described in `*_coordinatesystem.tsv` that the channel contains.                                                                                      |
+| name          | REQUIRED              | string        | Label of the channel. Entries have to match headers in (any) `*_motion.tsv.`                                                                                                                   |
+| tracked_point | REQUIRED              | string        | Label of the point that is being tracked, for example, label of a tracker or a marker (`LeftFoot`, `RightWrist`).                                                                              |
+| type          | REQUIRED              | string        | Type of data.                                                                                                                                                                                  |
+| units         | REQUIRED              | string        | Physical unit of the value represented in this channel, for example, vm for virtual meters, radian or degrees for angular quantities. See the BIDS spec for guidelines for Units and Prefixes. |
```

Comment:
I haven't seen virtual meters or degrees defined in the appendix for units: https://bids-specification.readthedocs.io/en/stable/99-appendices/05-units.html

```suggestion
| units         | REQUIRED              | string        | Physical unit of the value represented in this channel, for example, `vm` for virtual meters, `rad` radian or `deg` degrees for angular quantities. See the BIDS spec for guidelines for Units and Prefixes. |
```

Reply:
I agree, we included this as it can be used in virtual environments (for example VR). But I am happy to make this BIDS units complient.